src/components/FeatureAnnouncementBanner/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Alert } from '@edx/paragon';
+import { CheckCircle } from '@edx/paragon/icons';
+import { logError } from '@edx/frontend-platform/logging';
+
+import LmsApiService from '../../data/services/LmsApiService';
+
+const FeatureAnnouncementBanner = ({ enterpriseSlug }) => {
+  const [enterpriseNotificationBanner, setEnterpriseNotificationBanner] = useState({
+    text: null,
+    notificationId: null,
+  });
+  const [isRead, setIsRead] = useState(false);
+
+  const markAsRead = () => {
+    setIsRead(true);
+    LmsApiService.markBannerNotificationAsRead({
+      enterprise_slug: enterpriseSlug,
+      notification_id: enterpriseNotificationBanner.notificationId,
+    });
+  };
+
+  useEffect(() => {
+    LmsApiService.fetchEnterpriseBySlug(enterpriseSlug)
```

Comment:
We already fetch enterprise customer information when an admin selects an enterprise, can not we use that information instead of fetch it again?

Reply:
I tried but I do not think we can.

Reply:
The main  reason is that the component is rendered as a result of page routing and url only  contains enterprise slug and we must fetch all data from that slug and not depend on anything else.

Reply:
isn' t there any redux store or anything like that?